spec/index.html

```diff
@@ -225,30 +301,29 @@
           attribute boolean? expanded;
           attribute boolean? disabled;
           attribute InvalidType? invalid;
-          attribute boolean hasPopUp;
+          attribute boolean? hasPopUp;
+          attribute boolean? pressed;
 
           // Control values.
-          attribute DOMString? value;
+          attribute DOMString? valueText;
           attribute DOMString? placeholder;
-          attribute double? rangeValue;
-          attribute double? rangeMin;
-          attribute double? rangeMax;
-          attribute double? rangeStep;
+          attribute double? valueNow;
+          attribute double? valueMin;
+          attribute double? valueMax;
```

Comment:
Yeah, that's exactly the idea.

I guess some minor renaming could be okay, but not having a 1:1
correspondence significantly complicates this phase.